spec/counter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Counter do
+  describe '.initialize' do
+    it 'creates a counter' do
+      expect(Kitchen::Counter.new).to be_truthy
+    end
+  end
+  describe '.increment' do
+    counter = Kitchen::Counter.new
+    it 'increments by 1 given no args' do
+      val = counter.instance_variable_get(:@value)
+      counter.increment
+      expect(counter.instance_variable_get(:@value)).to eq(val + 1)
```

Comment:
Way to go on getting to the member variable -- advance! But per below, I think it is ok to just test based on `get`.